### ISSUE TITLE
Separate Apple Silicon/Intel download links on the app page

### ISSUE THREAD

**Issue by shreyasm-dev** [modrinth/code#1540]
### Describe the bug

On the app page, there are two different download links for macOS: one for Intel and one for Apple Silicon. However, both point to the same file, a universal binary that runs on both architectures, so this link separation is unnecessary.

### Steps to reproduce

1. Go to https://modrinth.com/app
2. Scroll down to the downloads

### Expected behavior

A single link specifying nothing or that it's a universal binary would suffice.

**Comment by Thinkseal** [comment]
I believe it is intentionally the same but having the option to download just one of them to reduce the download size would be nice

**Comment by Prospector** [comment]
yeah this is intentional, since when they were one link people would ask us when we would support arm or intel
